examples/dynalo/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# Copyright (c) 2018, Yassine Maddouri
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-dynalo)
+
+hunter_add_package(dynalo)
```

Comment:
Please use `DOCUMENTATION_START {` and `DOCUMENTATION_END }` here:
* https://github.com/ruslo/hunter/blob/78f356c878a42a3ace38ad8cf0599426563eebc0/examples/foo/CMakeLists.txt#L12-L18